src/contracts/aludel.ts

```diff
@@ -1,10 +1,14 @@
 import aludelAbi from "./aludelAbi";
 import { ethers } from "ethers";
-import { formatUnits } from "ethers/lib/utils";
+import { formatUnits, formatEther, parseEther } from "ethers/lib/utils";
 import IERC20 from "./IERC20.json";
+import { FixedNumber } from "ethers";
```

Comment:
I used the new FixedNumber format since some of the <1 ratio calculations weren't allowed by BigNumber.